Phase 2: Intro section (#9437)
open-telemetry/opentelemetry.io: content/en/docs/collector/introduction/_index.md

---

> title: Introduction

Reviewer: ```suggestion
title: Introduction to the OpenTelemetry Collector
```

> description: Everything You Need to Know About the Collector

Reviewer: ```suggestion
description: A vendor-neutral service for receiving, processing, and exporting telemetry
```

> aliases: [introduction]
> weight: 1

Reviewer: ```suggestion
weight: 1
```

cSpell:ignore: docker OTLP
---

<!-- markdownlint-disable ol-prefix blanks-around-fences -->

> The OpenTelemetry Collector offers a vendor-agnostic implementation of how to
> receive, process and export telemetry data. It removes the need to run, operate,
> and maintain multiple agents/collectors. This works with improved scalability
> and supports open source observability data formats (e.g. Jaeger, Prometheus,
> Fluent Bit, etc.) sending to one or more open source or commercial backends.
> 
> [What is the Collector?](what-is-the-collector/)

Reviewer: ```suggestion
The OpenTelemetry Collector is a flexible, vendor-neutral service for collecting, processing, and exporting telemetry. It can run as an agent or a centralized gateway in your observability pipeline. 

The Collector is scalable and extensible, and it works with many common telemetry protocols and formats. It allows you to send your telemetry to a variety of backends, including open source or commercial systems.

You can use the Collector in several ways:

- Not at all. Using the Collector is optional but recommended.
- As a vendor-neutral proxy, where the Collector only receives and exports data.
- As a proxy that also processes and transforms telemetry before exporting it.
```

![OpenTelemetry Collector diagram with Jaeger, OTLP and Prometheus integration](img/otel-collector.svg)

> ## Objectives
> 
> - _Usability_: Reasonable default configuration, supports popular protocols,
>   runs and collects out of the box.
> - _Performance_: Highly stable and performant under varying loads and
>   configurations.
> - _Observability_: An exemplar of an observable service.
> - _Extensibility_: Customizable without touching the core code.
> - _Unification_: Single codebase, deployable as an agent or collector with
>   support for traces, metrics, and logs.

Reviewer: ```suggestion
## Design principles

The OpenTelemetry Collector is designed with the following principles in mind:

- **Usability**: Provides sensible defaults, supports popular protocols,
  and can be run with minimal configuration.
- **Performance**: Handles production workloads efficiently and remains stable and performant under varying loads and configurations.
- **Observability** Exemplifies an observable service with built-in telemetry
- **Extensibility**: Can be extended with custom components without modifying the core codebase.
- **Unification**: Offers a single, consistent way to collect and process traces, metrics, logs, and profiles, whether deployed as an agent or a gateway.
```

> 
> ## When to use a collector
> 
> For most language specific instrumentation libraries you have exporters for
> popular backends and OTLP. You might wonder,
> 
> > under what circumstances does one use a collector to send data, as opposed to
> > having each service send directly to the backend?
> 
> For trying out and getting started with OpenTelemetry, sending your data
> directly to a backend is a great way to get value quickly. Also, in a
> development or small-scale environment you can get decent results without a
> collector.
> 
> However, in general we recommend using a collector alongside your service, since
> it allows your service to offload data quickly and the collector can take care
> of additional handling like retries, batching, encryption or even sensitive data
> filtering.
> 
> It is also easier to [setup a collector](quick-start) than you might think: the
> default OTLP exporters in each language assume a local collector endpoint, so if
> you launch a collector it will automatically start receiving telemetry.

Reviewer: We're going to move this to the "what is the collector page"

```suggestion

```

## Collector security

> Follow best practices to make sure your collectors are [hosted][] and

Reviewer: ```suggestion
Follow best practices to make sure your Collectors are [hosted][] and
```

[configured][] securely.

## Status

The **Collector** status is: [mixed][], since core Collector components
currently have mixed [stability levels][].

**Collector components** differ in their maturity levels. Each component has its
stability documented in its `README.md`. You can find a list of all available
Collector components in the [registry][].

Support is guaranteed for Collector software artifacts for a certain time period
based on the artifact's intended audience. This support includes, at minimum,
fixes for critical bugs and security issues. See the

> [support policies](https://github.com/open-telemetry/opentelemetry-collector/blob/main/VERSIONING.md)

Reviewer: ```suggestion
[support policies][]
```

for more details.

## Distributions

> Click here to learn [How to Choose a Distribution](choose-a-distribution/).

Reviewer: ```suggestion
The OpenTelemetry Collector offers five pre-built [distributions][] that combine Collector components to serve different needs. <!-- Learn [how to choose a distribution][] -->.
```

> [architecture]: /docs/collector/introduction/architecture
> [quick start]: /docs/collector/introduction/quick-start
> [what is the collector]: /docs/collector/introduction/what-is-the-collector
> [choose a distribution]: /docs/collector/introduction/choose-a-distribution
> [configured]: /docs/security/config-best-practices/
> [hosted]: /docs/security/hosting-best-practices/
> [latest release]:
> https://github.com/open-telemetry/opentelemetry-collector-releases/releases/latest
> [mixed]: /docs/specs/otel/document-status/#mixed
> [registry]: /ecosystem/registry/?language=collector
> [stability levels]:
> https://github.com/open-telemetry/opentelemetry-collector#stability-levels

Reviewer: ```suggestion
[configured]: /docs/security/config-best-practices/
[hosted]: /docs/security/hosting-best-practices/
[mixed]: /docs/specs/otel/document-status/#mixed
[registry]: /ecosystem/registry/?language=collector
[support policies]:
  https://github.com/open-telemetry/opentelemetry-collector/blob/main/VERSIONING.md
[stability levels]:
  https://github.com/open-telemetry/opentelemetry-collector#stability-levels
[distributions]: /docs/collector/distributions/
```
